packages/banner/src/external.scss

```diff
@@ -0,0 +1 @@
+@import "~hig-react/lib/hig-react.css";
```

Comment:
This imports the required CSS for `Icon`, `IconButton`, and `Text`. However, `hig-react` doesn't export the CSS for each component individually, so we're simply importing everything for now.

Additionally, this file is workaround for importing external CSS dependencies, as using `import 'hig-react/lib/hig-react.css';` in `src/index.js` fails to bundle.

These issues are actively being worked on.